src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -87,35 +83,39 @@ define(function (require, exports, module) {
         _setTooltipThrottled: null,
 
         /**
-         * Set of layer IDs with faces that have been mounted. Used to avoid
-         * rendering faces until they are first visible. Initialized when the
-         * panel is mounted.
+         * Set of layer IDs that is or was expanded. Used to avoid
+         * rendering child faces until they are first visible.
          *
          * @private
          * @type {Set.<number>}
          */
-        _mountedLayerIDs: null,
+        _expandedLayerIDs: new Set(),
+        
+        /**
+         * Set of layer IDs that are changed in the last document model update. This is updated when LayerPanel 
+         * receives a new "document" prop. See "LayerPanel#_layerDiff" for details.
+         */
+        _changedLayerIDs: new Set(),
```

Comment:
I believe that these properties end up in the prototype, so you shouldn't initialize it to a mutable value. Instead, you should initialize it in `componentWillMount` (or similar) so that each panel instance has it's own mutable set instance.